airflow/models/baseoperator.py

```diff
@@ -233,31 +232,28 @@ class derived from this one results in the creation of a task object,
         result
     :type do_xcom_push: bool
     """
-
     # For derived classes to define which fields will get jinjaified
-    template_fields = []  # type: Iterable[str]
+    template_fields: Iterable[str] = []
     # Defines which files extensions to look for in the templated fields
-    template_ext = []  # type: Iterable[str]
+    template_ext: Iterable[str] = []
     # Defines the color in the UI
     ui_color = '#fff'  # type: str
     ui_fgcolor = '#000'  # type: str
 
     pool = ""  # type: str
 
     # base list which includes all the attrs that don't need deep copy.
-    _base_operator_shallow_copy_attrs = ('user_defined_macros',
-                                         'user_defined_filters',
-                                         'params',
-                                         '_log',)  # type: Iterable[str]
+    BASE_OPERATOR_SHALLOW_COPY_ATTRS: Tuple[str, ...] = \
```

Comment:
Can you see if changing this variable (form removing the leading `_`) makes this show up in the TaskInstance details page in the Web UI?

Reply:
I'll revert it. Do not want to make some unnecessary changes.

Reply:
I reverted it back.